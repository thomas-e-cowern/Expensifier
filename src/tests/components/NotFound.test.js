import React from 'react';
import {
    shallow
} from 'enzyme';
import notFoundPage from '../../components/NotFound';

test('Should render NotFound page correctly', () => {
    
    const wrapper = shallow(<notFoundPage />);
    expect(wrapper).toMatchSnapshot();
});