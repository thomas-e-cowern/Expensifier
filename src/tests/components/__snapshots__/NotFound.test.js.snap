// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFound page correctly 1`] = `<notFoundPage />`;
